src/backend/utils/html/remove-zero-dot-braille.js

```diff
@@ -0,0 +1,13 @@
+module.exports = function (dom) {
+  const regexValue = new RegExp('u{2800}');
```

Comment:
To include unicode in JS Regex, you need to use `\u2800`.  Also, let's do it inline below, and use a global replace to get all instances:

```js
    if (!paragraphInnerHTML.replace(/\u2800/g, '').trim()) {
```